Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -1126,3 +1153,27 @@ private func shouldGoToLandingPage() -> Bool {
     return false
   }
 }
+
+private func emailVerificationCompletionData(_ data: Data?,
+                                             urlResponse: HTTPURLResponse?) -> (String, Bool)? {
+  guard let statusCode = urlResponse?.statusCode else { return nil }
+
+  switch statusCode {
+  case 200..<300:
+    let response = localizedString(
+      key: "Thanks_youve_successfully_verified_your_email_address",
+      defaultValue: "Thanks—you’ve successfully verified your email address."
+    )
+    return (response, true)
+  case 300..<Int.max:
```

Comment:
Might need to explicitly handle `403` here if we're displaying a message for an email being re-sent.

Reply:
Wouldn't the message being returned by the server handle all errors whether it's a `500` or specifically a `403` since it is a failure and also captured within the `300..<Int.max`?

Reply:
Yeah that seems right, so shouldn't need to handle it here.